src/components/Game/onMessage.ts

```diff
@@ -125,13 +120,13 @@ export const onMessage = (
         case "fold":
         case "allin":
           message.action = message.action + "_player";
+          // @todo do we need this ?
```

Comment:
@norbertdragan can you please advice me on what the below logic is for? It switches the value of the `gui_playerID` to the oponnent? So the next person to the left? Thank you 🙏  

```
if (message.gui_playerID == 0) {
     message.gui_playerID = 1;
     sendMessage(message, "player2", state, dispatch);
} else if (message.gui_playerID == 1) {
     message.gui_playerID = 0;
     sendMessage(message, "player1", state, dispatch);
}
```